### ISSUE TITLE
Sysroot asumptions

### ISSUE THREAD

**Issue by tegoo** [legatoproject/legato-af#3]
Not sure if it is a real issue.

Since Yocto 1.8 sysroot configuration is not hardcoded in generated SDK compiler so that 
`${TOOLCHAIN_PREFIX}-gcc --print-sysroot `
returns
`/not/exists`
So build scripts don't work with toolchain SDK built with newer Yocto.

**Comment by CoRfr** [comment]
Hi tegoo,

Thanks for the info, the toolchain management with Yocto < 1.8 has always been a bit painful (especially for the relocation ...).
We will try to solve this for next milestone but that's unsure at this point. Though if you already solved the issue feel free to do a pull request.

Thanks

**Comment by tegoo** [comment]
Hi CoRfr,

thanks for your reply. I have just created wrapper scripts for tools, which passes appropriate sysroot location:
```
#!/bin/sh

COMMAND=/opt/<mysystem>/1.0/sysroots/x86_64-<mysystem>sdk-linux/usr/bin/i586-<mysystem>-linux/i586-<mysystem>-linux-gcc-base
ARGS="--sysroot=/opt/<mysystem>/1.0/sysroots/i586-<mysystem>-linux"

exec $COMMAND "$ARGS" "$@"
```

**Comment by CoRfr** [comment]
Hi tegoo, Issue should be addressed in 16.04.0.
